flink-core/pom.xml

```diff
@@ -58,6 +58,12 @@ under the License.
 			<version>0.5.1</version>
 		</dependency>
 
+		<dependency>
```

Comment:
Actually, only a single convenience method in `AbstractID` depends on this. I would propose to refactor that method into one of the existing utility classes in `flink-runtime` and keep the Netty dependency in `flink-runtime`.